modules/standard/List.chpl

```diff
@@ -20,12 +20,14 @@
 /*
   This module provides a simple singly linked list.
 
-  .. note::
+  .. warning::
 
-      This module is expected to change in the future.
+      This module has been deprecated.
  */
 module List {
 
+compilerWarning("Module 'List' has been deprecated in favor of 'LinkedList'");
```

Comment:
In an effort to standardize deprecation messages (https://github.com/chapel-lang/chapel/issues/12438) - how do you feel about:

```chpl
compilerWarning("List module is deprecated - please use LinkedList instead");
```